### ISSUE TITLE
ARIA input fields do not have accessible names

### ISSUE THREAD

**Issue by cdrini** [internetarchive/openlibrary#9631]
This is an accessibility issue with the slick package which requires updating. The strategy for this one will be:

- Update our slick package to the latest version
- Replace static/css/lib/slick.css and static/css/lib/slick-theme.css with the CSS files from the slick package after updating.
- Test to make sure the carousels work correctly and no longer have the incorrect aria labels

**Comment by veronicamar** [comment]
From PageSpeed:
When an input field doesn't have an accessible name, screen readers announce it with a generic name, making it unusable for users who rely on screen readers.

<img width="901" alt="Screenshot 2024-07-27 at 2 49 05 pm" src="https://github.com/user-attachments/assets/83762d73-502c-43b4-8745-6bf24ceb046f">

<img width="463" alt="Screenshot 2024-07-27 at 2 50 04 pm" src="https://github.com/user-attachments/assets/f11af970-0052-4009-b076-ec096398218b">

**Comment by sanjanaynvsdl** [comment]
Hey @cdrini , @veronicamar please assign this issue to me.

**Comment by cdrini** [comment]
Hi @sanjanaynvsdl . This is an accessibility issue with the slick package which requires updating. The strategy for this one will be:

- Update our slick npm package to the latest version
- Replace static/css/lib/slick.css and static/css/lib/slick-theme.css with the CSS files from the slick package after updating.

Are you still interested?

**Comment by Spoorthy1423** [comment]
Hey, @cdrini i would love to work on this issue, can u please assign me this issue.

**Comment by mekarpeles** [comment]
@Spoorthy1423, sure give this a try locally if you'd like. Before we assign you, are there any questions you have about the task? Do you know how you will fetch the latest version of slick https://kenwheeler.github.io/slick/?

**Comment by lokesh** [comment]
The proposal to resolve this issue (from 2 years ago) is to upgrade Slick. We have a PR to rebuild the carousel with native scroll snapping https://github.com/internetarchive/openlibrary/pull/13220.  Upgrading Slick now might be throw-away work, so I would hold off.
